docs/contribution-guide/index.md

```diff
@@ -0,0 +1,7 @@
+---
+date: 2024-05-10
+category: document
+author: Revanth Rao Thopucherla
+order: 2
```

Comment:
I would put order higher so it's at the bottom of the sidebar

Reply:
done it will never come above anything now